packages/components-angular/projects/intranet-header-showcase/src/main.ts

```diff
@@ -3,6 +3,9 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
+import { defineCustomElements } from '@swisspost/design-system-components/loader';
+
+defineCustomElements();
```

Comment:
Do we really want to define all the custom Elements or only the post-icon component?

Reply:
Components are lazy loaded by default so there's no harm done here. Also, this is just the showcase for local testing.